pyramid_debugtoolbar/__init__.py

```diff
@@ -161,8 +161,8 @@ def make_application(settings, parent_registry):
     config.add_route('debugtoolbar.execute', '/execute')
     config.add_route('debugtoolbar.console', '/console')
     config.add_route(EXC_ROUTE_NAME, '/exception')
-    config.add_route('debugtoolbar.sql_select', '/sqlalchemy/sql_select')
-    config.add_route('debugtoolbar.sql_explain', '/sqlalchemy/sql_explain')
+    config.add_route('debugtoolbar.sql_select', '/{request_id}/sqlalchemy/sql_select')
+    config.add_route('debugtoolbar.sql_explain', '/{request_id}/sqlalchemy/sql_explain')
```

Comment:
I could do `/sqlalchemy/{request_id}/sql_select/{query_index}`, if you like.

Given that both parameters are required to be able to do anything, I think they both belong in the route.